styles/index.css

```diff
@@ -53,17 +53,16 @@ main {
   height: 48px;
   fill: #e85a4f;
   position: absolute;
-  right: 0;
-  top: 0;
-  margin-right: 20px;
-  margin-top: -15px;
+  right: 20px;
+  top: -15px;
 }
 
 .card--display-none {
   display: none;
 }
 ul {
   display: flex;
+  flex-wrap: wra;
```

Comment:
I think a p is missing here.

Reply:
and I was wondering why it's not working 😬